DataFormats/HeavyIonEvent/interface/EvtPlane.h

```diff
@@ -7,37 +14,41 @@
 
 #include <vector>
 #include <string>
+#include <cmath>
 
 namespace reco { class EvtPlane {
-public:
-   EvtPlane(double planeA=0,double sumSin=0, double sumCos=0,  std::string label="");
-  virtual ~EvtPlane();
-
-  double      angle()   const { return angle_; }
-  double      sumSin()  const { return sumSin_;}
-  double      sumCos()  const { return sumCos_;}
-  std::string label()   const { return label_; }
-
- 
-
-private:
-
-  double        angle_  ;
-  double        sumSin_;
-  double        sumCos_;
-  std::string   label_;
-
-
-};
-
- typedef std::vector<EvtPlane> EvtPlaneCollection;
-
+  public:
+    EvtPlane(int epindx=0, int level = 2, double planeA=0,double sumSin=0, double sumCos=0, double sumw = 0, double sumw2 = 0, double pe = 0, double pe2 = 0, uint mult = 0);
+    virtual ~EvtPlane();
+    void AddLevel(int level, double ang, double sumsin, double sumcos);
+    int indx() const { return indx_ ;}
+    float	angle(int level=2)   const { return (level>=0||level<=2)? angle_[level]:angle_[2]; }
```

Comment:
I don't understand the purpose of ``` (level>=0||level<=2) ``` condition, it's always true. Should this be && and in a different range, like ``` (level >=0 && level <4 ) ``` ?